### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by mxgross**

### CODE REVIEW

**Comment by kevinpapst** [comment]
Please include: https://www.kimai.org/documentation/export.html#adding-export-templates

**Comment by mxgross** [comment]
> Please include: https://www.kimai.org/documentation/export.html#adding-export-templates

Thanks fo ryour hint. Now I added var/export/ and templates/export/, subdirs of them are backuped also.

**Comment by kevinpapst** [comment]
Perfect 🚀

File: Resources/translations/system-configuration.en.xliff [hunk 1]
```diff
@@ -14,6 +14,10 @@
                 <source>label.easy_backup.setting_backup_dir</source>
                 <target>Backup directory</target>
             </trans-unit>
+            <trans-unit id="label.easy_backup.setting_paths_to_be_backuped">
+                <source>label.easy_backup.setting_paths_to_be_backuped</source>
+                <target>Files and paths to be backed up</target>
```
Comment (kevinpapst) [inline Resources/translations/system-configuration.en.xliff:19]: https://www.deepl.com/translator#de/en/Zu%20sichernde%20Dateien%20und%20Pfade
```suggestion
                <target>Files and paths to backup</target>
```

Comment (mxgross) [reply Resources/translations/system-configuration.en.xliff]: I changed it in the code as well. Thanks

**Pull request merged**